### ISSUE TITLE
can not alter table parameter like blocks.read_only_allow_delete

### ISSUE THREAD

**Issue by seanlook** [crate/crate#7471]
**CrateDB version**:
3.0.2 

**Problem description**:
only one node

**detail**

My disk is nearly full and writes is blocked. After I freed the data disk space, I need to reset the parameter `blocks.read_only_allow_delete`

refer: https://crate.io/docs/crate/reference/en/latest/sql/statements/create-table.html#id22

sql:
```
ALTER TABLE doc.metrics  RESET ("blocks.read_only_allow_delete");
ALTER TABLE doc.metrics  SET ("blocks.read_only_allow_delete" = false);
```

outputs:
```
Error!

SQLActionException[SQLParseException: Invalid property "blocks.read_only_allow_delete" passed to [ALTER | CREATE] TABLE statement]
```

I'm not sure it's my syntax error or it's a bug.

PS: When I execute `ALTER TABLE doc.metrics  RESET (number_of_replicas)`: 
```
Error!

SQLActionException[ClusterBlockException: blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];]
```
It doesn't report *Invalid property* error anyway.

**Comment by mfussenegger** [comment]
Thanks for reporting, we'll have a look at it

**Comment by mxm** [comment]
Thank you for reporting the issue. Regular tables are not affected by this bug, only partitioned tables are. I'm assuming your table is partitioned? 

A fix is pending for the upcoming release: #7485.

**Comment by mxm** [comment]
Fix will be included in the 3.0.3 release.

You can use the ES API in the meantime to set `index.blocks.read_only_allow_delete` on the indixes of the partitioned table. Let us know if you need help with that.
